Sources/SwiftKuery/ConnectionPool.swift

```diff
@@ -166,6 +167,29 @@ public class ConnectionPool {
         unlockPoolLock()
     }
     
+    private func increasePoolSize() -> Connection? {
```

Comment:
I don't like name or return type of this function.  This should be something like `tryToSatisfyMinimumPoolSize()` and should return a `Bool` indicating whether it was successful (ie. the pool is at the minimum size) or not.